sources/views/contacts/info.js

```diff
@@ -27,28 +41,71 @@ export default class ContactInfoView extends JetView {
 			const firstCol = {
 				rows: [
 					{view: "template", localId: "image", css: "contact-image", template: "Image"},
-					{view: "label", localId: "StatusID", label: "Status", css: "status", width: w}
+					{view: "label", localId: "StatusID", label: "", css: "status", width: w}
 				]
 			};
 			const secondCol = {
 				rows: [
-					{view: "label", localId: "Email", label: "Email", height: h, width: w, css: "label-info", template: this.createIconTemplate("fas fa-envelope")},
-					{view: "label", localId: "Skype", label: "Skype", height: h, width: w, css: "label-info", template: this.createIconTemplate("fab fa-skype")},
-					{view: "label", localId: "Job", label: "Job", height: h, width: w, css: "label-info", template: this.createIconTemplate("fas fa-tag")},
-					{view: "label", localId: "Company", label: "Company", height: h, width: w, css: "label-info", template: this.createIconTemplate("fas fa-building")}
+					{view: "label", localId: "Email", label: "", height: h, width: w, css: "label-info", template: this.createIconTemplate("fas fa-envelope")},
+					{view: "label", localId: "Skype", label: "", height: h, width: w, css: "label-info", template: this.createIconTemplate("fab fa-skype")},
+					{view: "label", localId: "Job", label: "", height: h, width: w, css: "label-info", template: this.createIconTemplate("fas fa-tag")},
+					{view: "label", localId: "Company", label: "", height: h, width: w, css: "label-info", template: this.createIconTemplate("fas fa-building")}
 				]
 			};
 			const thirdCol = {
 				rows: [
-					{view: "label", localId: "Birthday", label: "Date of birth", height: h, width: w, css: "label-info", template: this.createIconTemplate("fas fa-calendar-alt")},
-					{view: "label", localId: "Address", label: "Location", height: h, width: w, css: "label-info", template: this.createIconTemplate("fas fa-map-marker-alt")}
+					{view: "label", localId: "Birthday", label: "", height: h, width: w, css: "label-info", template: this.createIconTemplate("fas fa-calendar-alt")},
+					{view: "label", localId: "Address", label: "", height: h, width: w, css: "label-info", template: this.createIconTemplate("fas fa-map-marker-alt")}
 				]
 			};
 
-			const space = {
-				view: "template",
-				width: 50,
-				borderless: true
+			const space = {view: "template", width: 50, borderless: true};
+
+			this._tableActivities = new ActivitiesTableView(this.app, true);
+			const activitiesTab = {
+				id: "activitiesTab",
+				rows: [
+					this._tableActivities,
+					{view: "button", label: "Add activity", click: () => this._activityPopup.showPopup({ContactID: this._contactId})}
```

Comment:
Поле Contact не должно быть возможности поменять

> To do so, I will click on ‘Add activity’ button so that the form in popup window will appear on top of the table. In this form, I expect that the value in ‘contact’ field will match the currently selected contact and cannot be changed.